src/libpmem2/map.c

```diff
@@ -54,14 +54,35 @@ pmem2_get_length(const struct pmem2_config *cfg, size_t file_len,
 {
 	ASSERTne(length, NULL);
 
+	size_t alignment;
```

Comment:
I don't like that a function called `get` has a non-trivial logic that affects the output...